HelpMyStreet.Utils/HelpMyStreet.Contracts/RequestService/Request/UpdateRequestRequest.cs

```diff
@@ -0,0 +1,20 @@
+﻿using MediatR;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace HelpMyStreet.Contracts.RequestService.Request
+{
+    public class UpdateRequestRequest : IRequest
+    {
+        public int RequestID { get; set; }
+        public bool OnBehalfOfAnother { get; set; }
+        public bool HealthOrWellbeingConcern { get; set; }
+        public SupportActivityRequest SupportActivitiesRequired { get; set; }
+        public string FurtherDetails { get; set; }
+        public string RequestorFirstName { get; set; }
+        public string RequestorLastName { get; set; }
+        public string RequestorEmailAddress { get; set; }
+        public string RequestorPhoneNumber { get; set; }
```

Comment:
Why isn't there any validation on these properties (I appreciate this doesn't need be through attributes)?  Some of the columns in the DB are non-nullable so this could easily cause an error.  We probably also don't want to send an email without contact details.

There's a method in the ObjectExtensions class in the Address Service that can validate attributes.  This could be put in the NuGet package as Azure functions don't give you access to the ModelState you get in a normal MVC application.